app/models/types_de_champ/prefill_region_type_de_champ.rb

```diff
@@ -0,0 +1,15 @@
+class TypesDeChamp::PrefillRegionTypeDeChamp < TypesDeChamp::PrefillTypeDeChamp
+  def possible_values
```

Comment:
Plutôt que de les lister, on n'a pas un standard avec un lien wikipedia ou autre qu'on pourrait transmettre comme pour les pays?

Reply:
En réalité j'aime bien l'idée que ce soit l'application elle-même qui donne les valeurs qu'elle accepte, plutôt que d'avoir un lien vers une référence externe, qui peut changer et sur laquelle on n'a pas la main. Grâce à l'architecture des PrefillTYpeDeChamp, c'est assez facile à faire, et on pourrait faire de même pour les pays. D'autant qu'il y a un bug d'affichage à ce sujet : 

![image](https://user-images.githubusercontent.com/1193334/213652154-8c0697ea-7c17-431a-a807-8e1408bdf869.png)

Qu'en penses-tu ? Je ne suis pas figé, mais dans un sens ou dans l'autre, ça serait bien d'harmoniser notre approche.

Reply:
Je comprends ton point. Je trouve ça malgré tout vachement plus lourd en terme d'UX et moins claire de ne pas avoir de lien explicatif (pourquoi 32 c'est le Hauts-de-France?! Le lien m'expliquerait le concept de code région). Et j'ai suffisamment confiance dans un acteur externe comme Wikipédia pour être une référence externe de confiance. Mais ça se discute.

Par contre, dans tes valeurs, je me perds un peu car on a l'impression qu'il faut envoyer le numéro + le nom de la région. Heureusement que tu précises dans l'exemple en dessous.

Enfin, pour le problème d'affichage sur pays, il est introduit par la PR non? Il n'est pas sur main

Reply:
Qu'est-ce que tu penserais d'avoir TOUTES les infos ?

1/ Le lien Wikipédia (en l'occurrence [celui-ci](https://fr.wikipedia.org/wiki/R%C3%A9gion_fran%C3%A7aise), en expliquant que ce qui nous intéresse, c'est le code INSEE des régions)
2/ La liste des valeurs possibles, comme actuellement
3/ Un exemple de valeur possible, comme actuellement

Plutôt que Wikipédia, on peut aussi utilise le [Code Officiel Géographique](https://www.insee.fr/fr/information/6051727) de l'INSEE (mais ça oblige à fouiller d'avantage, alors c'est peut être pas top).

Reply:
Je crois que ça me parle:
- Dans valeur par défaut le lien wiki ou insee
- Au clic sur "voir les valeurs possibles" la liste actuelle des valeurs possibles
- Un exemple de valeur comme actuellement

Si ça te dit je suis ok pour faire ça sur tous les champs. ?Qui s'en occupe pour les pays?

Reply:
Je suis dessus. Je fais ça sur une nouvelle branche, car je corrige l'affichage du lien pour les pays (moi je l'ai bien sur main). Je te propose la PR tout à l'heure.

Reply:
En effet, je n'avais pas checké sur le dernier main. Sorry ! Une des dernières PR a dû le casser

Reply:
And voila :)

![image](https://user-images.githubusercontent.com/1193334/214061277-cec533cf-d96b-4c4c-b623-0f6057dca18a.png)